rrmngmnt/network.py

```diff
@@ -500,9 +500,9 @@ def send_icmp(self, dst, count="5", size="1500", extra_args=None):
         :return: True/False
         :rtype: bool
         """
-        cmd = ["ping", dst, "-c", count, "-s", size]
-        if size != "1500":
-            cmd.extend(["-M", "do"])
+        cmd = ["ping", dst, "-c", count]
+        if size and size.isdigit():
+            cmd.extend(["-s", "size", "-M", "do"])
```

Comment:
I believe that there should be `size` (without quotes)

Reply:
Oh yes, it is there by mistake... Thanks for noticing :)